packages/css-library/tokens/color.json

```diff
@@ -53,6 +53,9 @@
       "light": { "value": "#f9c642"},
       "lighter": { "value": "#fad980"},
       "lightest": { "value": "#fff1d2"}
+    },
+    "blue": {
+      "cool-lightest": { "value": "#dce4ef"}
```

Comment:
This color value is the same as "gray-cool-light", recommend using that color variable instead as it already exists.